docs/release-notes/Percona Everest 0.8.0 (2024-02-21).md

```diff
@@ -0,0 +1,21 @@
+# What's new in Percona Everest 0.8.0
+
+To begin your journey with Percona Everest, check out the [Quickstart Guide for Percona Everest](../quickstart-guide/quick-install.md).
+
+Percona Everest is an open source private database-as-a-service that helps developers deploy code faster, scale deployments rapidly, and reduce database administration overhead. Plus, you can regain control over your data, database configuration, and DBaaS costs.
+
+Version 0.8.0 introduces the following changes:
+
+## Bata release
+
+We're excited to announce that Percona Everest is now in Beta!
+
+Under development for the past six months and in testing since October 2023, we're now taking Percona Everest public, making it open for anyone who wants to explore our Cloud Native database platform.
+
+Be an early adopter and join us in driving its progress!
+
+## Support for scheduled backups for PostgreSQL
+
+We’re expanding Percona Everest’s schedule backup functionality to include PostgreSQL databases. This enables you to run automatic backups at predefined times for PostreSQL as well.
+
+However, due to a limitation on PostgreSQL Operator, scheduling is currently restricted to a maximum of three concurrent backups for PostgreSQL databases.
```

Comment:
1. "scheduling is currently restricted to a maximum of three concurrent backups" - confusing. "three concurrent backups" sounds like it's about running three backups simultaneously. But the restriction is about setting up no more than 3 backup _schedules_ 
2. I'm concerned if we should we specify the underlying technology. Wouldn't it more confusing for the user? Like - what's PostgreSQL operator? "However, due to a limitation on PostgreSQL Operator, scheduling is currently restricted to a maximum of three concurrent backups for PostgreSQL databases."